src/transformation/wavefront.rs

```diff
@@ -0,0 +1,42 @@
+use crate::shape::TriMesh;
+
+#[cfg(feature = "wavefront")]
+use obj::*;
```

Comment:
The glob-import should be replaced by importing each relevant member explicitly.